keyboards/coseyfannitutti/mulletpad/info.json

```
@@ -0,0 +1,11 @@
+{
+    "keyboard_name": "mulletpad", 
+    "url": "https://github.com/coseyfannitutti/mulletpad", 
+    "maintainer": "coseyfannitutti", 
+    "width": 4, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"label":"Num Lock", "x":0, "y":0}, {"label":"/", "x":1, "y":0}, {"label":"*", "x":2, "y":0}, {"label":"-", "x":3, "y":0}, {"label":"7", "x":0, "y":1}, {"label":"8", "x":1, "y":1}, {"label":"9", "x":2, "y":1}, {"label":"+", "x":3, "y":1, "h":2}, {"label":"4", "x":0, "y":2}, {"label":"5", "x":1, "y":2}, {"label":"6", "x":2, "y":2}, {"label":"1", "x":0, "y":3}, {"label":"2", "x":1, "y":3}, {"label":"3", "x":2, "y":3}, {"label":"Enter", "x":3, "y":3, "h":2}, {"label":"0", "x":0, "y":4, "w":2}, {"label":".", "x":2, "y":4}]
+        }
+    }
```

You're missing a closing curly bracket here.

```suggestion
    }
}
```